update.sh

```diff
@@ -85,15 +85,53 @@ for ext in "${exts[@]}"; do
 	fi
 
 	for algorithm in md5 sha1 sha256 sha512; do
-		${algorithm}sum "$archive" >> "$cwd/$ruby/checksums.$algorithm"
+		# 1) Append the new checksum line
+		"${algorithm}sum" "$archive" >> "$cwd/$ruby/checksums.$algorithm"
+
+		# 2) Remove duplicates (keeping the first occurrence) without sorting
+		awk '!seen[$0]++' "$cwd/$ruby/checksums.$algorithm" \
+			> "$cwd/$ruby/checksums.$algorithm.tmp"
+
+		mv "$cwd/$ruby/checksums.$algorithm.tmp" "$cwd/$ruby/checksums.$algorithm"
 	done
```

Comment:
Instead of using `awk` here to avoid duplicates, I considered using a similar sorting method as below; but it changed the default sort order of the file extensions/etc; and I didn't want to make too much diff churn here.

Also, currently this file seems to just have versions appended when they were added, not sorted by version number.. so I wasn't sure if that was just because it was easier or because we wanted this file to be roughly ordered by 'release date' rather than version number.

Reply:
Closing this comment thread in favour of https://github.com/postmodern/ruby-versions/pull/86/files#r1924611377